Azure/azure-cli
# [ARM] `az resource list`: Extend the return data of `createdTime`, `changedTime` and `provisioningState`

Issue: #14598

**Description**  

Extend the return data of `createdTime`, `changedTime` and `provisioningState` for `az resource list`.
These fields are returned before, but there is no value. Now make up these values.
REST API has supported: [link](https://docs.microsoft.com/en-us/rest/api/resources/resources/list#genericresourceexpanded)

**Testing Guide**  

Execute command `az resource list` to see whether `createdTime`, `changedTime` and `provisioningState` returns

**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
